framework/PageFramework.css

```diff
@@ -38,7 +38,11 @@
 }
 
 :root:where(.pf-v5-theme-dark) .bg-lighten {
-  background-image: linear-gradient(#ffffff08, #ffffff08);
+  background-image: linear-gradient(#ffffff08, #ffffff08) !important;
```

Comment:
I don't like using `!important` here... in my testing it works as expected without it